.travis.yml

```diff
@@ -6,6 +6,10 @@ notifications:
 sudo: false
 script:
 - npm test
+before_install:
+- openssl aes-256-cbc -K $encrypted_ca4a154c2e6b_key -iv $encrypted_ca4a154c2e6b_iv -in config/tenon.json.enc -out config/tenon.json -d
+before_deploy:
+- test $TRAVIS_TEST_RESULT = 0
```

Comment:
If $TRAVIS_TEST_RESULT = 0 then the build is successful.
This means we need a successful build before we deploy.